toonz/sources/toonzqt/fxschematicnode.cpp

```diff
@@ -357,7 +286,7 @@ void FxColumnPainter::contextMenuEvent(QGraphicsSceneContextMenuEvent *cme) {
 
   QAction *collapse = CommandManager::instance()->getAction("MI_Collapse");
 
-  QAction *openSubxsh = new QAction(tr("&Open Subxsheet"), &menu);
+  QAction *openSubxsh = new QAction(tr("&Open Sub-xsheet"), &menu);
   connect(openSubxsh, SIGNAL(triggered()), fxScene, SLOT(onOpenSubxsheet()));
```

Comment:
It seems that you can replace these two lines by
`QAction *openSubxsh = CommandManager::instance()->getAction("MI_OpenChild");`

And you can remove the function `FxSchematicScene::onOpenSubxsheet()`.

Reply:
Done